client/flutter/lib/main.dart

```diff
@@ -98,13 +118,12 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
-    // TODO: This is not essential for basic operation but we should implement 
-    // Fires if notification settings change. 
-    // Modify user opt-in if they do. 
-    // _firebaseMessaging.onIosSettingsRegistered
-    //     .listen((IosNotificationSettings settings) {
-    // });
-
+  // TODO: This is not essential for basic operation but we should implement
+  // Fires if notification settings change.
+  // Modify user opt-in if they do.
+  // _firebaseMessaging.onIosSettingsRegistered
```

Comment:
Think we need to implement this now - since we're requesting tokens after the initial permission.

Reply:
made an issue for this -- #902